packages/model/src/gp2/working-group.ts

```diff
@@ -15,6 +15,16 @@ export type WorkingGroupMember = {
   avatarUrl?: string;
 };
 
+const workingGroupResourceType = ['Link', 'Note'] as const;
+export type WorkingGroupResourceType = typeof workingGroupResourceType[number];
+
+export type WorkingGroupResource = {
```

Comment:
as per https://github.com/yldio/asap-hub/blob/87ad25cbbd8dccb272e13c00126eb84eaa5dcf79/apps/gp2-server/test/data-providers/working-group.data-provider.test.ts#L199
the WorkingGroup data provider should not return externalLinks for the WorkingGroup of type 'Note' and I believe your types should reflect it. 

I wouldn't mind having the data provider return whatever shit comes out squidex but since you are already making sure this doesn't happen why not have it typed correctly.